frontend/app/chat/components/ChatsList/index.tsx

```diff
@@ -13,14 +13,30 @@ export function ChatsList() {
 
   return (
     <MotionConfig transition={{ mass: 1, damping: 10 }}>
-      <div className="lg:sticky fixed top-0 left-0 bottom-0 overflow-visible z-30">
+      <motion.div
+        drag="x"
+        dragConstraints={{ right: 0, left: 0 }}
+        // dragSnapToOrigin
+        dragElastic={0.15}
+        onDragEnd={(event, info) => {
+          if (info.offset.x > 100 && !open) {
+            setOpen(true);
+          } else if (info.offset.x < -100 && open) {
+            setOpen(false);
+          }
+        }}
+        className="lg:sticky fixed top-0 left-0 bottom-0 overflow-visible z-30 border-r border-black/10 dark:border-white/25 bg-white dark:bg-black"
+      >
         <motion.div
-          animate={{ width: open ? "fit-content" : 0 }}
-          className={cn("overflow-hidden", {
-            "shadow-lg": open,
-          })}
+          animate={{
+            width: open ? "fit-content" : "0px",
+            opacity: open ? 1 : 0.5,
+            boxShadow: open ? "none" : "10px 10px 16px rgba(0, 0, 0, 0.5)",
+            // shadow: open ? "none" : "10px 10px 16px black",
```

Comment:
We can remove unused comments in the future :D

It would be even better if we could find an ESLint rule to prevent them.